react/features/mobile/watchos/reducer.js

```diff
@@ -0,0 +1,43 @@
+import { assign, ReducerRegistry, set } from '../../base/redux';
+import {
+    SET_CONFERENCE_TIMESTAMP,
+    SET_CONFERENCE_URL,
+    SET_MIC_MUTED,
+    SET_RECENT_URLS
+} from './actionTypes';
+
+const INITIAL_STATE = {
+    // NOTE for some reason 'null' does not update context
+    conferenceURL: 'NULL',
+    micMuted: false,
+    recentURLs: []
+};
+
+/**
+ * Reduces the Redux actions of the feature features/recording.
+ */
+ReducerRegistry.register(
```

Comment:
do we really need this reducer? looks like all the deta can be fetched from other pieces of state like recent-list and base/media.

Reply:
the entire reducer is sent as a context to the watch plus it has few extra fields that can not be fetched directly

Reply:
what fields? I think we could reconstruct the context by reading from different feature states, no?

Reply:
from extra fields there is `sessionID` and `conferenceTimestamp`

Reply:
plus it would not be simpler nor easier to try to do that

Reply:
Gotcha, but I don't want to keep duplicated state. Some `getContext(state)` function could pick the recents list stuff from the recents-list feature, muted from base/media, and these 2 from here.